force-app/main/default/lwc/geGift/geGift.js

```diff
@@ -45,6 +45,15 @@ class Gift {
             softCredits: { ...this._softCredits }
         }
     }
+
+    isRemovableFromElevateBatch() {
+        // PLACEHOLDER; validate that Status = 'Authorized' AND ...
+        return true;
+    }
+
+    isGiftAuthorized() {
```

Comment:
Could we rename this to `isAuthorized`?

Reply:
Absolutely! I had made that change myself but just hadn't committed yet. Glad to see we're aligned.